moped-database/migrations/1670022156862_alter_table_public_features_add_column_is_deleted/up.sql

```diff
@@ -0,0 +1,3 @@
+alter table "public"."features" add column "is_deleted" boolean
```

Comment:
Added `is_deleted` on the `features` table so that we can look for it by its unique ID and id of the component it is related to. We also don't have to be concerned about what specific feature table the record exists in.